### ISSUE TITLE
Including USGS CoNED DEM data base

### ISSUE THREAD

**Issue by saeed-moghimi-noaa** [noaa-ocs-modeling/OCSMesh#13]
@SorooshMani-NOAA 

To include USGS tile indexes to donlowd tiles from the site in case NOAA sites were missing.

See here:
https://topotools.cr.usgs.gov/topobathy_viewer/dwndata.htm

**Comment by SorooshMani-NOAA** [comment]
@saeed-moghimi-noaa currently OCSMesh doesn't have any auto-download capabilities. We could write a small script based on the needs and slowly integrate it into OCSMesh. For example I wrote a simple script that fetches NCEI tiles. We could extend that and polish the code and add it as a capability to the OCSMesh, what do you think?

**Comment by saeed-moghimi-noaa** [comment]
@SorooshMani-NOAA 

Yes, that what I meant. Lets develop that script as a utility / tool for now. 

Thanks,
-Saeed
